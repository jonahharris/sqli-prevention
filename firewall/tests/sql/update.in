>>> Sada testovacich SQL dotazu: update
UPDATE MyGuests SET lastname='Doe' WHERE id=2;
UPDATE LOW_PRIORITY MyGuests SET lastname='Doe' WHERE id=2;
UPDATE IGNORE MyGuests SET lastname='Doe' WHERE id=2;
UPDATE LOW_PRIORITY IGNORE MyGuests SET lastname='Doe' WHERE id=2;
UPDATE IGNORE MyGuests SET lastname = DEFAULT WHERE id=2;
UPDATE IGNORE MyGuests SET MyGuests.lastname = DEFAULT WHERE id=2;
UPDATE tab1, tab2 SET column1 = DEFAULT, column2 = DEFAULT WHERE tab1.id = tab2.id;
UPDATE tab1, tab2 SET tab1.column1 = DEFAULT, tab1.column2 = DEFAULT WHERE tab1.id = tab2.id;
UPDATE table_name SET column1 = value1, column2 = value2 WHERE id=100;
UPDATE tab1, tab2 SET tab1.column1 = value1, tab1.column2 = value2 WHERE tab1.id = tab2.id;
UPDATE LOW_PRIORITY tab1, tab2 SET tab1.column1 = value1, tab1.column2 = value2 WHERE tab1.id = tab2.id;
UPDATE IGNORE tab1, tab2 SET tab1.column1 = value1, tab1.column2 = value2 WHERE tab1.id = tab2.id;
UPDATE LOW_PRIORITY IGNORE tab1, tab2 SET tab1.column1 = value1, tab1.column2 = value2 WHERE tab1.id = tab2.id;
UPDATE tableA,tableB SET tableA.color=tableB.color WHERE tableA.id=tableB.id;
UPDATE table_name SET column1 = value1, column2 = value2 ORDER BY id;
UPDATE table_name SET column1 = value1, column2 = value2 LIMIT 2;
UPDATE table_name SET column1 = value1, column2 = value2 WHERE id=100 ORDER BY id LIMIT 25;
UPDATE seq SET x='z' WHERE x='b' ORDER BY i DESC LIMIT 1;
