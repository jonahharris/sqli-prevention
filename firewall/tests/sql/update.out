>>> Sada testovacich SQL dotazu: update
STRING '***';ASSIGN lastname;NAME id;NUMBER 2;CMP 4;WHERE;TABLE MyGuests;UPDATE ONE 0;STMT;1;0;
STRING '***';ASSIGN lastname;NAME id;NUMBER 2;CMP 4;WHERE;TABLE MyGuests;UPDATE ONE 1;STMT;1;0;
STRING '***';ASSIGN lastname;NAME id;NUMBER 2;CMP 4;WHERE;TABLE MyGuests;UPDATE ONE 2;STMT;1;0;
STRING '***';ASSIGN lastname;NAME id;NUMBER 2;CMP 4;WHERE;TABLE MyGuests;UPDATE ONE 3;STMT;1;0;
DEFAULT;ASSIGN lastname;NAME id;NUMBER 2;CMP 4;WHERE;TABLE MyGuests;UPDATE ONE 2;STMT;1;0;
DEFAULT;ASSIGN MyGuests.lastname;NAME id;NUMBER 2;CMP 4;WHERE;TABLE MyGuests;UPDATE ONE 2;STMT;1;0;
TABLE tab1;TABLE tab2;DEFAULT;ASSIGN column1;DEFAULT;ASSIGN column2;FIELDNAME tab1.id;FIELDNAME tab2.id;CMP 4;WHERE;UPDATE MULTI 0 2 2;STMT;1;0;
TABLE tab1;TABLE tab2;DEFAULT;ASSIGN tab1.column1;DEFAULT;ASSIGN tab1.column2;FIELDNAME tab1.id;FIELDNAME tab2.id;CMP 4;WHERE;UPDATE MULTI 0 2 1;STMT;1;0;
NAME value1;ASSIGN column1;NAME value2;ASSIGN column2;NAME id;NUMBER 100;CMP 4;WHERE;TABLE table_name;UPDATE ONE 0;STMT;1;0;
TABLE tab1;TABLE tab2;NAME value1;ASSIGN tab1.column1;NAME value2;ASSIGN tab1.column2;FIELDNAME tab1.id;FIELDNAME tab2.id;CMP 4;WHERE;UPDATE MULTI 0 2 1;STMT;1;0;
TABLE tab1;TABLE tab2;NAME value1;ASSIGN tab1.column1;NAME value2;ASSIGN tab1.column2;FIELDNAME tab1.id;FIELDNAME tab2.id;CMP 4;WHERE;UPDATE MULTI 1 2 1;STMT;1;0;
TABLE tab1;TABLE tab2;NAME value1;ASSIGN tab1.column1;NAME value2;ASSIGN tab1.column2;FIELDNAME tab1.id;FIELDNAME tab2.id;CMP 4;WHERE;UPDATE MULTI 2 2 1;STMT;1;0;
TABLE tab1;TABLE tab2;NAME value1;ASSIGN tab1.column1;NAME value2;ASSIGN tab1.column2;FIELDNAME tab1.id;FIELDNAME tab2.id;CMP 4;WHERE;UPDATE MULTI 3 2 1;STMT;1;0;
TABLE tableA;TABLE tableB;FIELDNAME tableB.color;ASSIGN tableA.color;FIELDNAME tableA.id;FIELDNAME tableB.id;CMP 4;WHERE;UPDATE MULTI 0 2 1;STMT;1;0;
NAME value1;ASSIGN column1;NAME value2;ASSIGN column2;NAME id;ORDERBY 0;ORDERBYLIST 1;TABLE table_name;UPDATE ONE 0;STMT;1;0;
NAME value1;ASSIGN column1;NAME value2;ASSIGN column2;NUMBER 2;LIMIT 1;TABLE table_name;UPDATE ONE 0;STMT;1;0;
NAME value1;ASSIGN column1;NAME value2;ASSIGN column2;NAME id;NUMBER 100;CMP 4;WHERE;NAME id;ORDERBY 0;ORDERBYLIST 1;NUMBER 25;LIMIT 1;TABLE table_name;UPDATE ONE 0;STMT;1;0;
STRING '***';ASSIGN x;NAME x;STRING '***';CMP 4;WHERE;NAME i;ORDERBY 2;ORDERBYLIST 1;NUMBER 1;LIMIT 1;TABLE seq;UPDATE ONE 0;STMT;1;0;
